sql/parser/sql.go

```diff
@@ -333,313 +333,319 @@ var yyExca = []int{
 	-2, 0,
```

Comment:
I think Github uses some library to determine which diffs to render, and that (among other things) looks at comments usually found at the top of auto-generated files. They won't have one for us, but maybe one specific enough that we can abuse unless we want to PR it upstream.
@bdarnell knows for sure.